manifests/c/cristiano100/roforge/0.1/cristiano100.roforge.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: cristiano100.roforge
+PackageVersion: "0.1"
+InstallerType: exe

```

Comment:
```suggestion
InstallerType: portable
```
This seems like a portable package, since running the installer appears to launch the full application. Is this correct?
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/60a82ac8-1e7d-4649-a006-7daa9032888d)